cmds/bmp/tasks_command.go

```diff
@@ -51,5 +55,41 @@ func (cmd tasksCommand) Validate() (bool, error) {
 
 func (cmd tasksCommand) Execute(args []string) (int, error) {
 	cmd.printer.Printf("Executing %s comamnd: args: %#v, options: %#v", cmd.Name(), args, cmd.options)
+
+	var latest uint
+	latest = 50
```

Comment:
As far as I know, it seems golang doesn't have optional parameter for functions.
http://stackoverflow.com/questions/2032149/optional-parameters